src/ScheduledTasks/Exception/ScheduleAlreadyExistException.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.DurableTask.ScheduledTasks;
+
+/// <summary>
+/// Exception thrown when attempting to create a schedule with an ID that already exists.
+/// </summary>
+public class ScheduleAlreadyExistsException : InvalidOperationException
```

Comment:
Where is this used? I couldn't find any references.

Reply:
since we do createorupdate or createifnotexists, this is not needed anymore, will remove